docs/src/developing/clients/jsonrpc-api.md

```diff
@@ -206,11 +206,11 @@ Returns all information associated with the account of provided Pubkey
 fields:
   - (optional) [Commitment](jsonrpc-api.md#configuring-state-commitment)
   - `encoding: <string>` - encoding for Account data, either "base58" (*slow*),
-"base64", or jsonParsed". "base58" is limited to Account data of less than 128
+"base64", or "jsonParsed". "base58" is limited to Account data of less than 128
 bytes. "base64" will return base64 encoded data for Account data of any size.
-Parsed-JSON encoding attempts to use program-specific state parsers to return
+"jsonParsed" encoding attempts to use program-specific state parsers to return
```

Comment:
Do you mind making this change everywhere `Parsed-JSON` is used, for consistency?